### DISCUSSION TITLE
Added OTFAD driver (and other adaptions) to Zephyr

### DISCUSSION THREAD

**Discussion by lam-borg** [zephyrproject-rtos/zephyr#51426] [Show and tell] [answered]
I prepared a pull request that will add the OTFAD driver and also add XBARA and ADC_ETC to Kconfig for the imx rt116x und 117x
This pull only applies to our zephyr fork. I'll probably bring it upstream later.

Could one of the pros have a look at that?
https://github.com/mathias-landolt-loepfe/zephyr/pull/1

As I wrote in the comment of the pull request, I don't get why the edit it the file `soc/arm/nxp_imx/rt/Kconfig.soc` didn't do the trick and I had to add it here `soc/arm/nxp_imx/rt/Kconfig.defconfig.mimxrt1176_cm7`

Whats the reason for the RT1176 section in Kconfig.soc if it's not used or overwritten?

**Comment by DerekSnell** [comment, marked as answer]
Hi @mathias-landolt-loepfe ,
Are you referring to selecting the XBARA feature?  It looks like your kconfig.soc line 338 has a typo.  `HAS_MCUX_XBAR` should be `HAS_MCUX_XBARA`
